src/index.js

```diff
@@ -3,8 +3,8 @@ import ReactDOM from 'react-dom';
 import '@fortawesome/fontawesome/styles.css';
 import App from './App';
 import './shared/styles/normalize.global.css';
-import './shared/styles/react-table.global.css';
 import './shared/styles/react-select.global.css';
+import './shared/styles/notifications.css';
```

Comment:
@kylemh Should we also remove the file `/public/notification.css` ?

Another thing I'm not too knowledgeable about (possibly due to my inexperience w/ Webpack) is why we have two index HTML files - a `/public/index.html` and root-level `index.html`.  In the public file, I still see the older (non-defer) Raven scripts, and also `<link rel="stylesheet" href="%PUBLIC_URL%/notification.css">`.  Might we want to apply some of this PR's changes there too?  On my dev machine, running this branch, I'm still seeing a CSS network request made for `notifications.css`.